Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1562,69 +1565,12 @@ private void CaptureInitialState()
             var target = Target;
             if (target != null)
             {
-                initialScaleAtStart = target.transform.localScale;
-
-                maximumScale = initialScaleAtStart * scaleMaximum;
-                minimumScale = initialScaleAtStart * scaleMinimum;
-                isChildOfTarget = transform.IsChildOf(target.transform);
```

Comment:
>isChildOfTarget [](start = 16, length = 15)

This line isn't getting set anymore, but still exists elsewhere - should this remain?

Reply:
Oops, I'll add that back.